guide/creating-your-bot/slash-commands.md

```diff
@@ -138,4 +138,4 @@ You can implement additional commands by creating additional files in the `comma
 
 #### Resulting code
 
-<ResultingCode path="creating-your-bot/slash-commands" />
\ No newline at end of file
+<ResultingCode path="creating-your-bot/slash-commands" />
```

Comment:
unrelated change

Reply:
Removed extra line. (It was from prettier plugin)

Reply:
still shows up as edited line in the diff

Reply:
Still not resolved _(very visually confusing that you are using my discord avatar on GitHub, btw)_

Reply:
> Still not resolved _(very visually confusing that you are using my discord avatar on GitHub, btw)_

Somehow, Github ends up adding an extra line to the commit. I don't know why. _(Oh that snake guy?)_